NuGet.Docs/ndocs/API/Visual-Studio-Credential-Providers.md

```diff
@@ -0,0 +1,76 @@
+﻿# NuGet Credential Providers for Visual Studio
+
+The NuGet Visual Studio Extension 3.6+ now supports Credential Providers, which enable NuGet to work seamlessly with authenticated feeds.
+After you install a NuGet Credential Provider for Visual Studio, the NuGet Visual Studio extension will automatically acquire and refresh credentials for authenticated feeds as necessary.
+
+Note: NuGet Credential Providers for Visual Studio only work in Visual Studio (not in dotnet restore or nuget.exe). To learn more about NuGet.exe Credential Providers, check out [this page](NuGet.exe-Credential-Providers.md).
+
+NuGet Credential Providers for Visual Studio must be installed as a regular Visual Studio extension.
+
+## Available NuGet Credential Providers for Visual Studio
+
+There is a single credential provider that is built into the Visual Studio NuGet extension to support Visual Studio Team Services.
+
+The NuGet Visual Studio Extension uses an internal `VsCredentialProviderImporter` which also scans for plug-in credential providers. These plug-in credential providers must be discoverable as a MEF Export of type `IVsCredentialProvider`.
+
+## Creating a NuGet Credential Provider for Visual Studio
+
+The NuGet Visual Studio Extension 3.6+ implements an internal CredentialService that is used to acquire credentials. The CredentialService has a list of built-in and plug-in Credential Providers. Each provider is tried sequentially until credentials are acquired.
+
+During credential acquisition, the credential service will try credential providers in the following order, stopping as soon as credentials are acquired:
+
+a) Credentials will be fetched from NuGet configuration files (using the built-in `SettingsCredentialProvider`).
```

Comment:
a)
b)
 ....
 is not recognized as a list in markdown. consider  using a numbered list 1. 2. ....